modules/enable-monitoring-central-operator.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="enable-monitoring-central-operator_{context}"]
+= Monitoring Central using the {product-title-short} Operator
+
+[role="_abstract"]
+You can monitor Central, by changing the Central configuration options using the {product-title-short} Operator.
```

Comment:
```suggestion
You can monitor {product-title-short} services by changing the configuration of the `Central` custom resource.
```

While `using the operator` is technically correct, I feel it more precise to refer to the custom resource here. Also enabling scanner monitoring via "Central configuration" sounds a bit weird, so I'd avoid that phrase.